module3/homework/advanced_calculator/test.cpp

```diff
@@ -124,9 +124,9 @@ TEST(advancedCalculatorTest, ShouldFactorial) {
     ASSERT_EQ(process("5!", &result), ErrorCode::OK);
     EXPECT_TRUE(cmp(result, 120));
     ASSERT_EQ(process("6.7!", &result), ErrorCode::OK);
```

Comment:
What does factorial mean on a floating point numbers? IMO there should be no such operation possible